### PR TITLE
potential 2.4 dep viewer cherry picks

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

cherry picking latest dependency viewer updates.
Removing one API I don't think is required. @QilongTang 

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@QilongTang 

### FYIs

@smangarole

### CODE REVIEW

File: test/pkgs/Dynamo Samples/extra/CustomRenderExample.dyn [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      ]
+    }
+  ],
+  "Bindings": [],
+  "View": {
+    "Dynamo": {
+      "ScaleFactor": 1.0,
+      "HasRunWithoutCrash": true,
+      "IsVisibleInDynamoLibrary": true,
+      "Version": "2.5.0.5915",
```
Comment (QilongTang) [inline test/pkgs/Dynamo Samples/extra/CustomRenderExample.dyn]: Can you change it to basic 2.4.0?

**Comment by mjkkirschner** [comment]
@QilongTang updated - PTAL

File: src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs [hunk 2]
```diff
@@ -5,8 +5,7 @@
 using System.Windows.Controls;
 using Dynamo.Controls;
 using Dynamo.Extensions;
-using Dynamo.Logging;
-using Dynamo.Selection;
+=using Dynamo.Selection;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs]: @mjkkirschner  There is an extra "=" here causing the build to fail

**Comment by QilongTang** [comment]
One last comment, then LGTM. Once the checks are good, feel free to merge. Would love to get new RC localized and bang on it together.

File: src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs (hunk 2 above)
Comment (mjkkirschner) [reply src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs]: thanks!

**Comment by QilongTang** [comment]
LGTM

**Pull request merged**